static/js/theme-map/ThemeMapConfig.js

```diff
@@ -142,13 +142,13 @@ export default class ThemeMapConfig {
      */
     this.enablePinClustering = jsonConfig.enablePinClustering;
 
-    const noResultsConfig = jsonConfig.noResults || {};
+    const noResultsConfig = jsonConfig.noResultsConfig || {};
```

Comment:
did we change the name in the config from noResults to noResultsConfig?

Reply:
Nope! Was always `noResultsConfig`, we just were referencing the wrong name!
https://github.com/yext/answers-hitchhiker-theme/blob/release/v1.20/static/js/theme-map/VerticalFullPageMapOrchestrator.js#L233